yaml-tests/src/test/resources/aggregate-index-tests.yamsql

```diff
@@ -278,23 +269,25 @@ test_block:
       - result: [{COL3: 1, S: 1}, {COL3: 2, S: 2}, {COL3: 100, S: 1}, {COL3: 200, S: 2}]
     -
       - query: select col3, sum(col2) as s from t2 use index (mv9) where col1 = 1 group by col1, col3 order by col3 asc;
-      # Cannot use FORCE_CONTINUATIONS with older versions due to: https://github.com/FoundationDB/fdb-record-layer/issues/3096
-      # (Extra values being produced after exhausting source of an aggregate cursor)
-      # Can remove once we do not care about backwards compatibility before 4.1.9.0
-      - maxRows: 0
-      - result: [{COL3: 1, S: 1}, {COL3: 2, S: 2}, {COL3: 100, S: 1}, {COL3: 200, S: 2}]
+      - maxRows: 1
```

Comment:
I don't think we need to add `maxRows: 1` here. In general, we can just remove the `maxRows: 0` clause, and then the framework will generate the force-continuations stuff for us. In this case, this is another instance where we already have such a test case (in this case, the immediately previous query), and we should be able to just remove the `supported_version` line from there and then delete this now extraneous test case